app/dv/src/test/resources/generated-pom.xml

```diff
@@ -68,7 +68,7 @@
       <plugin>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-maven-plugin</artifactId>
-        <version>2.2.4.RELEASE</version>
+        <version>2.2.6.RELEASE</version>
```

Comment:
Perhaps a better solution would be to use property placeholders so we don't need to update this next time?

Like we do here:

https://github.com/syndesisio/syndesis/blob/d07d800359a2cc6b3adea449f4472f01038db701/app/integration/project-generator/src/main/resources/io/syndesis/integration/project/generator/templates/pom.xml.mustache#L24